ci/coverage_publish.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo "Coverage"
+echo $(pwd)
+
+echo "NOTE: envoy build dir $ENVOY_BUILD_DIR"
+ls -laR $ENVOY_BUILD_DIR
```

Comment:
Drive by review:

* Generally prefer `"$ENVOY_BUILD_DIR"` to avoid whitespace in path issues. 

* Best to set `set -e` above.